tests/serverpod_test_server/migrations/migration_registry.txt

```diff
@@ -5,4 +5,4 @@
 ### the conflict by removing and recreating the conflicting migration.
 
 20240125141253049
-20240827080004999
+20240827093851370
```

Comment:
Suggestion: Remove the recreated migration.

There is no difference in the migrations for exceptions (since they are not stored in the database.)